calico_containers/pycalico/datastore.py

```diff
@@ -336,6 +340,14 @@ def add_ip_pool(self, version, pool):
         assert version in (4, 6)
         assert isinstance(pool, IPPool)
 
+        # If IP in IP is enabled, ensure that it is enabled globally, and if
+        # not enable it.
```

Comment:
The "and if not enable it" sounds funny, I think it can just be removed since ensuring something is enabled implies you will enable it if necessary.